custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, month):
     def child_temp_tablename(self):
         return get_child_health_temp_tablename(self.month_start)
 
+    @property
+    def agg_child_temp_tablename(self):
+        return get_agg_child_temp_tablename()
```

Comment:
may be this function can be directly used. without the property function.

Reply:
I generally prefer this style, because I like having everything encompassed in the class, but either would work. Although the property method is easier to use in f strings than a function